packages/firestore/test/integration/api/query.test.ts

```diff
@@ -2061,34 +2062,113 @@ apiDescribe('Queries', (persistence: boolean) => {
     });
   });
 
-  // TODO(Mila): Skip the test when using emulator as there is a bug related to 
-  // sending existence filter in response: b/270731363. Remove the condition  
-  // here once the bug is resolved.
+  // TODO(b/270731363): Re-enable this test to run against the Firestore
+  // emulator once the bug where an existence filter fails to be sent when a
+  // query is resumed is fixed.
   // eslint-disable-next-line no-restricted-properties
-  (USE_EMULATOR ? it.skip : it)(
-    'resuming a query should remove deleted documents indicated by existence filter',
-    () => {
+  (USE_EMULATOR || !persistence ? it.skip : it)(
```

Comment:
Remove `!persistence` and instead include it in the "if" statement that checks to whether or not to verify the bloom filter.